src/go/pdwfs.go

```diff
@@ -267,7 +272,7 @@ func Write(fd int, buf []byte) int {
 
 	n, err := (*file).Write(buf)
 
-	check(err) // no known conversion to errno, just panic if err != nil
+	// check(err) // no known conversion to errno, just panic if err != nil
```

Comment:
we shouldn't return an error code (n<0) without properly setting errno, this will make user code hard to debug.
I think it's best to crash for now

Reply:
I will handle error with readonly and set errno accordingly (C.EBADF).
With other errors the application will crash.